### DISCUSSION TITLE
How can I `vc dev` without a deployment?

### DISCUSSION THREAD

**Discussion by nathanchu** [vercel/vercel#5488] [Help] [answered]
I'm working on a project that uses vercel. The git deployment is owned by another vercel account. When I started working on it I had to create a pretty much useless deployment on my account just so I could develop locally. Is this the only way? Or is there a way I can `vc dev` without a deployment on vercel.com?

**Comment by TooTallNate** [comment, marked as answer]
There's the `vc link` command which you can run first, instead of creating a deployment. This will create a Project on your account, but won't create the useless deployment.

**Reply by nathanchu** [reply]
Thanks, I didn't know about that. :+1:
